chem_spectra/controller/spectra_layout_api.py

```diff
@@ -0,0 +1,25 @@
+import json
+from flask import jsonify, Blueprint
+import os
+import shutil
+from flask_jwt_extended import jwt_required
+
+spectra_layout_api = Blueprint('spectra_layout_api', __name__)
+script_dir = os.path.dirname(__file__)
+data_type_json_path = os.path.join(script_dir, '../lib/converter/jcamp/data_type.json')
+
+def load_data_types():
+    try:
+        with open(data_type_json_path, 'r') as mapping_file:
+            return json.load(mapping_file)
+    except FileNotFoundError:
+        example_json_path = os.path.join(script_dir, '../lib/converter/jcamp/data_type.json.example')
+        shutil.copy(example_json_path, data_type_json_path)
+        with open(data_type_json_path, 'r') as mapping_file:
+            return json.load(mapping_file)
+
+@spectra_layout_api.route('/api/v1/chemspectra/spectra_layouts', methods=['GET'])
+@jwt_required()
```

Comment:
What will happen if jwt token is expired, how we renew and exchange the new token?